roles/etcd_client_certificates/meta/main.yml

```diff
@@ -14,3 +14,4 @@ galaxy_info:
   - system
 dependencies:
 - role: etcd_common
+- role: delegated_serial_command
```

Comment:
> edit note that this could be interpreted as going against the recommended directory layout per http://docs.ansible.com/ansible/playbooks_best_practices.html#directory-layout

That guide doesn't take into account that modules and other plugins can be embedded inside of roles. Ops really likes using roles to allow better re-use and in discussions with some Ansible folks about this, they didn't see any issues with this method. Requiring modules that exist outside of a role would also preclude being able to publish usable roles to Galaxy.

All that said, now that we are on 2.x, we should move our filter plugins into (a) role(s)